C.js

```diff
@@ -0,0 +1,76 @@
+let taskNumber = 1;
+const toDoList = document.getElementById("toDo");
+const doneList = document.getElementById("done");
+
+const createNewTask = () => {
+    const task = document.createElement("div");
+    task.id = "t" + taskNumber;
+    task.classList.add("toDoTask", "task");
+
+    return task;
+}
+
+const createNewCheckbox = () => {
+    const checkbox = document.createElement("input");
+    checkbox.type = "checkbox";
+    checkbox.addEventListener("click", switchTaskList);
+    
+    return checkbox;
+}
+
+const createTaskDescription = (event) => {
+    const taskDescription = document.createElement("span");
+    taskDescription.innerText = event.target.value;
+
+    return taskDescription;
+}
+
+const createDeleteTaskButton = () => {
+    const deleteTaskButton = document.createElement("button");
+    deleteTaskButton.innerText = "X";
+    deleteTaskButton.classList.add("deleteButton");
+    deleteTaskButton.addEventListener("click", deleteTask);
+
+    return deleteTaskButton
+}
+
+const newTask = (event) => {

```

Comment:
you already have `createNewTask`, what `newTask` does?
please chose more indicative names